docs/source/releases/40_0.rst

```diff
@@ -12,7 +12,7 @@ The major changes introduced on this version are listed below.
 
 * The introduction of a tool that generated a diff-like report of two
   jobs.  For more information on this feature, please check out its
-  own :ref:`documentation <job_diff_>`.
```

Comment:
Why are there references being removed?

Reply:
Same here.

Since that many refs were changed, the plan was to add back the references on a second phase.

I removed to get rid of the warning messages.